### DISCUSSION TITLE
Coin stack value filter to ground items plugin

### DISCUSSION THREAD

**Discussion by DelVision** [runelite/runelite#17289] [Ideas]
So this is relatively simple suggestion, it would be nice if you could filter ground items plugin to only show coin stacks above specified value..

Coin drops tend to fill the screen very often with nearly worthless drops of 1-1000 coins so it would be nice if you could set a custom amount in ground items plugin that will be hidden without needing to toggle the coins on and off depending on the monster you're killing.

**Comment by raiyni** [comment]
You can do that by doing `coins<1001`

**Reply by DelVision** [reply]
thank you, I'll close this discussion then.. Didn't know you could do that
